lib/mixlib/install/generator/bourne/scripts/helpers.sh

```diff
@@ -210,10 +210,6 @@ do_checksum() {
     echo "Comparing checksum with sha256sum..."
     checksum=`sha256sum $1 | awk '{ print $1 }'`
     return `test "x$checksum" = "x$2"`
-  elif exists shasum; then
-    echo "Comparing checksum with shasum..."
-    checksum=`shasum -a 256 $1 | awk '{ print $1 }'`
-    return `test "x$checksum" = "x$2"`
```

Comment:
Actually I misread this code @patrick-wright. It wasn't doing `SHA1` checks but using `shasum` to validate the `SHA256`. We def want to leave this clause in there for compatibility on Unixes that might now have the `sha256sum` binary installed.